lms/validation/authentication/_bearer_token.py

```diff
@@ -148,13 +150,13 @@ def _decode_jwt(self, data, **_kwargs):
 
         https://marshmallow.readthedocs.io/en/2.x-line/extending.html#example-enveloping
         """
-        try:
-            jwt = data["authorization"][len("Bearer ") :]
-        except KeyError:
+        if data["authorization"] == marshmallow.missing:
             raise marshmallow.ValidationError(
                 "Missing data for required field.", "authorization"
             )
 
+        jwt = data["authorization"][len("Bearer ") :]
+
```

Comment:
There is a [note lower down in this file](https://github.com/hypothesis/lms/blob/e0c410af3b1751c1dce81a271860b53687a01c70/lms/validation/authentication/_bearer_token.py#L190) to remove a workaround once an upstream issue is closed, and that issue is now closed. Does the comment need to be updated or the workaround removed?

Reply:
It seems we can simply delete that now, yes. I'll add a commit to do so